css/src/_base.scss

```diff
@@ -68,6 +68,14 @@ a {
     }
 }
 
+button {
+    border: none;
+    border-radius: 6px;
+    padding: 10px 15px;
+    margin: 0 20px 44px 0;
+    font-weight: bold;
+}
+
```

Comment:
Not all buttons will follow this style. These styles should be defined within the `.m-btn` module.